### PR TITLE
🔨 Add _NCS#keyboardInput

### PR DESCRIPTION
**Pull request by Florian-1801**

### CODE REVIEW

File: src/api/players/client/keyboardInput.lua [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+---@param TextEntry string
+---@param ExampleText string
+---@param MaxStringLenght number
+function _NCS:keyboardInput(TextEntry, ExampleText, MaxStringLenght)
+
+    AddTextEntry('FMMC_KEY_TIP1', TextEntry)
+
+    blockinput = true
+    DisplayOnscreenKeyboard(1, "FMMC_KEY_TIP1", "Somme", ExampleText, "", "", "", MaxStringLenght)
+    while UpdateOnscreenKeyboard() ~= 1 and UpdateOnscreenKeyboard() ~= 2 do
```
Comment (PABLO-1610) [inline src/api/players/client/keyboardInput.lua:11]:
```suggestion
    while (UpdateOnscreenKeyboard() ~= 1 and UpdateOnscreenKeyboard() ~= 2) do
```

File: src/api/players/client/keyboardInput.lua [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        Wait(0)
+    end
+
+    if UpdateOnscreenKeyboard() ~= 2 then
```
Comment (PABLO-1610) [inline src/api/players/client/keyboardInput.lua:15]:
```suggestion
    if (UpdateOnscreenKeyboard() ~= 2) then
```

File: src/api/players/client/keyboardInput.lua [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        local result = GetOnscreenKeyboardResult()
+        Wait(500)
+        blockinput = false
+        return result
```
Comment (PABLO-1610) [inline src/api/players/client/keyboardInput.lua:19]:
```suggestion
        return (result)
```

File: src/api/players/client/keyboardInput.lua [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    else
+        Citizen.Wait(500)
+        blockinput = false
+        return nil
```
Comment (PABLO-1610) [inline src/api/players/client/keyboardInput.lua:23]:
```suggestion
        return (nil)
```

**Review by PABLO-1610 — CHANGES_REQUESTED** [review]
Please respect the code conventions 😀

**Comment by Florian-1801** [comment]
Yes, mybaad, I think is better like that

**Review by PABLO-1610 — APPROVED** [review]

**Comment by PABLO-1610** [comment]
In the future @Florian-1801, please try to respect the name of the commits with https://gitmoji.dev/.

**Pull request merged**